packages/cloudflare/src/api/cloudflare-context.ts

```diff
@@ -45,6 +45,14 @@ type InternalGlobalThis<
   __NEXT_DATA__: Record<string, unknown>;
 };
 
+type GetCloudflareContextOptions = Partial<{
```

Comment:
Do we need the Partial here?

Reply:
No we don't, I was just future proofing this object in case in the future there might be more options, I can totally remove the `Partial` if you prefer (it does make little sense when there is only a single option)

Reply:
Yeah, we can add it later if needed

Reply:
removed 👍